admin/src/components/Avatar/styles.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-nocheck
```

Comment:
Please do not create new files with `@ts-nocheck`. We want to incrementally introduce as much ts with each commit as possible.